content/videos/teachable-machine/3-sound-classification/index.json

```diff
@@ -0,0 +1,80 @@
+{
+  "title": "Teachable Machine 3: Sound Classifiication",
+  "description": "Introducing Teachable Machine 2.0 (from Google Creative Lab)!  Train a computer to recognize your own images, sounds, & poses. In this video, I train a sound classifier and import the machine learning model into a p5.js sketch with the ml5.js library",
+  "videoNumber": "3",
+  "videoId": "TOrVsLklltM",
+  "date": "2019-11-11",
+  "languages": ["JavaScript", "p5.js", "ml5.js"],
+  "topics": ["machine learning", "ml5","sound classification", "snake", "game"],
+  "canContribute": true,
```

Comment:
Should we allow showcase contributions on all 3 videos?

Reply:
i have changed them all to true now.